examples/error/src/main/scala/410.scala

```diff
@@ -0,0 +1,31 @@
+package ticket410
+
+trait T[A]
+
+class C[A](a: A) extends T[Int] {
+  def this() = this(???)
+}
+
+class D extends C(1)
```

Comment:
This case is also a problem, which currently isn't caught because it uses a `TptSelect` instead of a `TptId`:
`class D2 extends ticket410.C(1)`

In general, if the first parent is a `TptPath` instead of a `TptParameterize`, we need to check it. I think the only valid forms of `TptPath` for a class parent are `TptId` and `TptSelect`, since `TptProject` and `TptSingleton` aren't valid in this context, and are caught by `scalac`.

Reply:
Thanks for the additional details